src/module/rules/AttributeRules.ts

```diff
@@ -13,18 +11,23 @@ export class AttributeRules {
      * Also implements the 'use bigger value rule',if necessary.
      *
      * @param names A list of attribute names to inject
-     * @param attributes The list of source attribute to pull from
+     * @param source The Actor or Item to use as the source of attributes
      * @param rollData The testData to inject attributes into
      * @param options.bigger If true, the bigger value will be used, if false the source value will always be used.
      */
-    static injectAttributes(names: readonly string[], attributes: AttributesType | TechnologyAttributesType, rollData: SR5Actor['system'] | SR5Item['system'], options: { bigger: boolean }) {
+    static injectAttributes(names: readonly string[],
+                            source: SR5Actor | SR5Item,
+                            rollData: SR5Actor['system'] | SR5Item['system'],
+                            options: { bigger: boolean }) {
```

Comment:
I'd do the following, so it uses the same structure used in other parts of the code:
```ts
    static injectAttributes(
        names: readonly string[],
        source: SR5Actor | SR5Item,
        rollData: SR5Actor['system'] | SR5Item['system'],
        options: { bigger: boolean }
    ) {
```